File: lovely-frontend/src/app/interfaces/profileDetailDTO.ts
export interface ProfileDetailDTO {
    id: number;
    phone: string;
    gender: string;
    birthDate: Date; // Puedes manejar la fecha como string o convertirla según tus necesidades
    description: string;
    work: string;
    photo: File | null;
    photoFileName: string;
    timestamp: string; // Puedes manejar la fecha como string o convertirla según tus necesidades
  }
  

File: lovely/src/main/java/com/nl/lovely/dto/ProfileDetailDTO.java
package com.nl.lovely.dto;

import java.time.LocalDate;
import java.time.LocalDateTime;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ProfileDetailDTO {
	 	 Long id;
		 String phone;
	     String gender;
	     LocalDate birthDate;
	     String description;
	     String work;
	     byte[] photo;
	     String photoFileName;
	     LocalDateTime timestamp;
}
